src/ips-nmos-api/src/http/subscription/add-token.js

```diff
@@ -0,0 +1,30 @@
+module.exports = () => {
+  const TOKEN_KEY = 'accessToken'
+  if (!window.sessionStorage) {
+    console.error('Please use a browser that supports Session Storage')
+    return
+  }
+
+  return {
+    fetch () {
+      if (window.sessionStorage.getItem(TOKEN_KEY)) {
+        let accessToken = window.sessionStorage.getItem(TOKEN_KEY)
+        return accessToken
+      }
+    },
+    addAuthHeaders (headers) {
+      let accessToken = this.fetch()
+      if (accessToken) {
+        let authString = `Bearer ${accessToken}`
+        if (headers) headers['Authorization'] = authString
```

Comment:
Style point. Generally I don't see many peeps doing single line if statements in js.

Mostly due to these kinda bugs https://medium.com/@jonathanabrams/single-line-if-statements-2565c62ff492